mutmut/__main__.py

```diff
@@ -5,26 +5,29 @@
 
 import itertools
 import os
+import shlex
+import subprocess
 import sys
 import signal
 from datetime import datetime
 from functools import wraps
 from io import open
 from os.path import isdir, exists
 from shutil import move, copy
-from subprocess import Popen
-from threading import Thread
-from time import sleep
+from threading import Timer
 
 import click
 from glob2 import glob
+from nonblock import nonblock_read
```

Comment:
That lib doesn't work on mac while the existing popen_streaming_output does. So no go.

Reply:
Thanks for the heads up, saves me some testing time!